resources/js/AppRoutes.jsx

```diff
@@ -1,6 +1,7 @@
 import { BrowserRouter, Route, Routes } from "react-router-dom";
+import CampaignOverview from "./routes/campaign/CampaignOverview";
 import Connections from "./routes/connections/Connections";
-import Dashboard from "./routes/dashboard/Dashboard";
+import MainProfile from "./routes/dashboard/MainProfile";
```

Comment:
I think these might need to be separate routes, the profile page shouldn't make api calls about available followers or mass campaigns, and I'm thinking we might want to even make this a public page that does not require authentication? Do one of those like instagram type UX where you can see this page, but if you click on anything we ask you to login/signup?

From a performance standpoint I see that loading the profile hits the mass follower and available follow endpoints and it should only really hit the Get User by Username endpoint once.